src/test/java/alinatinca/tests/Tema6/HerokuForms.java

```diff
@@ -0,0 +1,189 @@
+    public static void firstParagraph() {
+        WebElement firstParagraph = driver.findElement(By.className("explanation"));
+        //find the first paragraph
+        System.out.println("The first paragraph has the following message: " + firstParagraph.getText());
+        //display the message for the first paragraph
+    }
+
+    public static void scrollToSubmitButton() {
+        WebElement submitButton = driver.findElement(By
+                    .xpath("//input[@type='submit']"));
+        //find the submit button
+        Actions actions = new Actions(driver);
+        //create an object of type Actions
+        actions.moveToElement(submitButton).build().perform();
+        //scroll to the submit button
+        System.out.println("Scroll to the submit button");
+        //display the above message
+    }
+
+    public static void fillFormDetails(){
+        WebElement userName = driver.findElement(By.cssSelector("input[name='username']"));
+        //find the Username field
+        WebElement password = driver.findElement(By.cssSelector("input[name='password']"));
+        //find the Password field
+        WebElement textAreaComment = driver.findElement(By.cssSelector("textarea[name='comments']"));
+        //find the TextArea Comment field
+        WebElement fileUpload = driver.findElement(By.cssSelector("input[name='filename']"));
+        //find the Filename field
+        List <WebElement> checkBoxes = (List<WebElement>) driver.findElements(By.name("checkboxes[]"));
+        //find the list for the Checkbox Items
+        List <WebElement> radioButtons = (List<WebElement>) driver.findElements(By.name("radioval"));
+        //find the list for the Radio Items
+        Select multipleSelectValue = new Select(driver.findElement(By.name("multipleselect[]")));
+        //find the list for Multiple Select Values
+        Select dropdown = new Select(driver.findElement(By.name("dropdown")));
+        //find the list for Dropdown
+        WebElement submitButton = driver.findElement(By.cssSelector("input[type='submit']"));
+        //find the submit button
+        userName.click();
+        //click on Username field
+        userName.sendKeys("test_demo");
+        //fill the Username field with the above text
+        userName.sendKeys(Keys.TAB);
+        //Go to the next field with TAB
+        password.click();
+        //click on Password field
+        password.sendKeys("password_test");
+        //fill the Password field with the above text
+        password.sendKeys(Keys.TAB);
+        //Go to the next field with TAB
+        textAreaComment.click();
+        //click on TextArea Comment field
+        textAreaComment.clear();
+        //clear the TextArea Comment field
+        textAreaComment.sendKeys("comment_test");
+        //fill the TextArea Comment field with the above text
+        textAreaComment.sendKeys(Keys.TAB);
+        //Go to the next field with TAB
+        fileUpload.sendKeys("E:\\Alina\\Screenshot.png");
```

Comment:
Try to avoid using local paths when adding files. This won't work on other machines. Create the file inside the repo, in your package for homework 6 and get the path from there. Thanks!